apisix/plugins/prometheus/exporter.lua

```diff
@@ -80,9 +80,14 @@ local function extra_labels(name, ctx)
         for _, kv in ipairs(labels) do
             local val, v = next(kv)
             if ctx then
-                val = ctx.var[v:sub(2)]
-                if val == nil then
-                    val = ""
+                if v:find("$",1, true) then
```

Comment:
Will "a$b" match?

Reply:
not, "a$b" will be considered as a const now

Reply:
@xinxinh2020 
This branch will match `a$b`.

Reply:
``` 
￥ cat /tmp/x.lua
#!/usr/bin/env luajit

local v = "a$b"
print(v:find("$",1, true))

 ￥ /tmp/x.lua
2       2
```

Reply:
> @xinxinh2020 This branch will match `a$b`.

Thinks for reminding.  I fix it already.